### DISCUSSION TITLE
Callback is ignored with filepath property in response

### DISCUSSION THREAD

**Discussion by darknet-lord** [mockoon/mockoon#1284] [Help]
Hi! I created a simple endpoint with a callback. It's workes fine with "inline" and "data" responses. But if I switch the response type to "file" my callback stops working (the response itself is working correctly). Is this normal or am I doing something wrong? Thanks for the help!

**Comment by 255kb** [comment]
Hi, you are right it's a bug! I've created a [new issue](https://github.com/mockoon/mockoon/issues/1285). It's a quick fix that will be in the January release.

**Reply by darknet-lord** [reply]
Thank you!
